model/src/main/scala/za/co/absa/atum/model/utils/SerializationUtils.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.model.dto.utils
+
+import org.json4s.jackson.Serialization
+import org.json4s.jackson.Serialization.{write, writePretty}
+import org.json4s.{Formats, NoTypeHints, ext}
+import za.co.absa.atum.model.dto.MeasureResultDTO.ResultValueType
+
+import java.time.format.DateTimeFormatter
+
+object SerializationUtils {
+
+  implicit private val formatsJson: Formats =
+    Serialization.formats(NoTypeHints).withBigDecimal + new ext.EnumNameSerializer(ResultValueType) +
+      ext.UUIDSerializer + OffsetDateTimeSerializer
+
+  val timestampFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSSX")
+
+  /**
+   * The method returns arbitrary object as a Json string.
+   *
+   * @return A string representing the object in Json format
+   */
+  def asJson[T <: AnyRef](obj: T): String = {
```

Comment:
**For future:**
We might consider to use some value class for Json strings (existing one or our own).

Reply:
Good point, let's address it when we introduce value classes.